packer.json

```diff
@@ -0,0 +1,105 @@
+{
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "region": "us-east-1",
+      "source_ami": "ami-840910ee",
+      "instance_type": "t2.micro",
+      "ssh_username": "ubuntu",
+      "ami_name": "lint-review-{{timestamp}}",
+      "tags": {
+        "Name": "lint-review-{{timestamp}}",
+        "OS": "ubuntu"
+      },
+      "ami_description": "Lint review ami created {{timestamp}}",
+      "associate_public_ip_address": true,
+      "run_tags": {
+        "Name": "Packer Lint Review AMI builder"
+      }
+    }
+  ],
+  "provisioners": [
+    {
+      "type": "file",
+      "source": "package.json",
+      "destination": "/home/ubuntu/package.json"
+    },
+    {
+      "type": "file",
+      "source": "requirements.txt",
+      "destination": "/home/ubuntu/requirements.txt"
+    },
+    {
+      "type": "file",
+      "source": "composer.json",
+      "destination": "/home/ubuntu/composer.json"
+    },
+    {
+      "type": "file",
+      "source": "composer.lock",
+      "destination": "/home/ubuntu/composer.lock"
+    },
+    {
+      "type": "file",
+      "source": "Gemfile",
+      "destination": "/home/ubuntu/Gemfile"
+    },
+    {
+      "type": "file",
+      "source": "Gemfile.lock",
+      "destination": "/home/ubuntu/Gemfile.lock"
+    },
+    {
+      "type": "file",
+      "source": "systemd/celery.service",
+      "destination": "/home/ubuntu/celery.service"
+    },
+    {
+      "type": "file",
+      "source": "systemd/gunicorn.service",
+      "destination": "/home/ubuntu/gunicorn.service"
+    },
+    {
+      "type": "file",
+      "source": "settings.sample.py",
+      "destination": "settings.py"
+    },
+    {
+      "type": "file",
+      "source": "logging.ini",
+      "destination": "logging.ini"
+    },
+    {
+      "type": "shell",
+      "execute_command": "sudo -E sh -e '{{ .Path }}'",
+      "inline": [
+        "DEBIAN_FRONTEND=noninteractive apt-get update",
+        "DEBIAN_FRONTEND=noninteractive apt-get upgrade -y",
+        "DEBIAN_FRONTEND=noninteractive apt-get update",
```

Comment:
yeah, the apt-get install fails without the update after the upgrade, and the update doesn't work is the update doesn't happen first :-/